pkgs/by-name/ma/maskprocessor/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maskprocessor";
+  version = "0.73";
+
+  src = fetchFromGitHub {
+    owner = "hashcat";
+    repo = "maskprocessor";
+    tag = "v${version}";
+    hash = "sha256-LVtMz5y0PbKKuc92W5xW0C84avigR7vS1XL/aXkUYe8=";
+  };
+
+  # upstream Makefile is terrible, this simplifies everything.
+  buildPhase = ''
+    $CC -o maskprocessor src/mp.c -W -Wall -std=c99 -O2 -s -m64 -DLINUX
+  '';
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  doInstallCheck = true;
+
+  installPhase = ''
+    install -Dm544 -t $out/bin maskprocessor
+  '';
+
+  meta = {
+    homepage = "https://github.com/hashcat/maskprocessor";
+    description = "High-Performance word generator with a per-position configureable charset.";
```

```suggestion
    description = "High-Performance word generator with a per-position configureable charset";
```

See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md

---

Will fix.